packages/lib/src/components/Card/Card.tsx

```diff
@@ -66,7 +65,7 @@ export class CardElement extends UIElement<CardElementProps> {
             // billingAddressRequired only available for non-stored cards
             billingAddressRequired: props.storedPaymentMethodId ? false : props.billingAddressRequired,
             // ...(props.brands && !props.groupTypes && { groupTypes: props.brands }),
-            type: props.type === 'scheme' ? 'card' : props.type,
+            type: TxVariants.card,
```

Comment:
Do we still need the `TxVariant.card` ? What is it used for? I expected that we would need only `TxVariant.scheme`

Reply:
`this.props.type` (with value `TxVariant.card`) is an internal property that propagates down to the securedFields to tell them what "type" they are.
All the securedFields code is built around being a "card"

Reply:
Ah, ok.. seems to be a massive change then 😅